pkg/network/netbinding/netbinding.go

```diff
@@ -88,10 +93,16 @@ func SlirpNetBindingPluginSidecar(vmi *v1.VirtualMachineInstance, kvConfig *v1.K
 		slirpSidecarImage = plugin.SidecarImage
 	}
 
-	return hooks.HookSidecar{
-		Image:           slirpSidecarImage,
-		ImagePullPolicy: kvConfig.ImagePullPolicy,
+	var pluginSidecars hooks.HookSidecarList
+	for _, iface := range slirpIfaces {
```

Comment:
The sidecar is generic and not interface specific. Therefore, we should have only one sidecar per plugin, even if there a multiple interfaces using the same plugin/binding.
Seems the previous (refactored) code had the same bug...